cli/storage_s3.go

```diff
@@ -25,6 +25,7 @@ func (c *storageS3Flags) Setup(svc StorageProviderServices, cmd *kingpin.CmdClau
 	cmd.Flag("prefix", "Prefix to use for objects in the bucket").StringVar(&c.s3options.Prefix)
 	cmd.Flag("disable-tls", "Disable TLS security (HTTPS)").BoolVar(&c.s3options.DoNotUseTLS)
 	cmd.Flag("disable-tls-verification", "Disable TLS (HTTPS) certificate verification").BoolVar(&c.s3options.DoNotVerifyTLS)
+	cmd.Flag("rootca", "Certficate authority (base64 enc.)").StringVar(&c.s3options.RootCA)
```

Comment:
Can we call the flag root-ca-base64?

curious why not take the name of the file and read it and base64 encode ourselves?

we can support either option.

Reply:
Renamed as `root-ca-pem-base64`, can be feed via env using `ROOT_CA_PEM_BASE64`
File option implemented with `root-ca-pem-path`.

My first approach was file based, then I thought it would be nice to be able to inline it for automation purpose (kube for example, I don't say it's not doable via file but inline-ing with env is easier with `$()` syntax, still an example).
Now both are handled